ACKategoriesCore/RandomExtensions/Randomizable.swift

```diff
@@ -0,0 +1,20 @@
+import Foundation
+
+/// Mark a type Randomizable
+/// to get automatically synthesized `.random()` method for `Collection`s
+public protocol Randomizable {
+    /// - Returns: Random instance of `Self`
+    static func random() -> Self
+}
+
+public extension Collection where Element: Randomizable {
+    /// - Parameters:
+    ///     - min: Minimal number of returned elements
+    ///     - max: Maximal number of returned elements
+    /// - Returns: Array of random elements
+    static func random(min: Int = 20, max: Int = 100) -> [Element] {
+        assert(min <= max, "Min must be smaller than max")
+        let count = (Int.random() % max) + min
+        return (0..<count).map { _ in .random() }
+    }
+}
```

Comment:
This code is flawed for a case when `min = max`. In that case, one can get a wrong number of results (up to `2 * max - 1`).
I'd also suggest you create a test for this (edge) case.

Reply:
Used `Int.random(in: ...)`, which should fix it, good catch 👍 
Negative values are not allowed anymore (it will fail in `(0..<count)`)